src/main/java/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> reports = new HashMap<>();
```

Comment:
The 'reports' Map should be private to follow the encapsulation principle. Exposing it as public allows for uncontrolled access and modification from anywhere in the code, which can lead to bugs and makes the code less maintainable.